runtime/v1/shim/service.go

```diff
@@ -589,6 +571,27 @@ func (s *Service) forward(publisher events.Publisher) {
 	}
 }
 
+// getProcess returns initial process or exec process
+func (s *Service) getProcess(id string) (rproc.Process, error) {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+
+	switch id {
+	case "init":
```

Comment:
`init` is used by moby I guess.

Reply:
You could do this as an empty string, however I think it might just make sense to have another function with a different signature, like `getInitProcess()`

Reply:
+1

Reply:
"init" string is I want to distinguish `s.id` and `r.ID`, consider that `r.ID` also can be empty.  I agree 
 with to make `getInitProcess()`  to do this, since this is also my first considered plan. It not make confused  with `init`.